tests/gcp/test-utils_gcp.R

```diff
@@ -0,0 +1,194 @@
+tar_test("gcp_gcs_exists()", {
+  skip_if_no_gcp()
+  auth_gcp()
+  bucket <- random_bucket_name()
+  # needs to be a GCP project the tester auth has access to
+  projectId <- Sys.getenv("GCE_DEFAULT_PROJECT_ID")
+
+  googleCloudStorageR::gcs_create_bucket(bucket, projectId = projectId)
+  on.exit(gcp_gcs_delete_bucket(bucket))
+  expect_false(gcp_gcs_exists(key = "x", bucket = bucket))
+  tmp <- tempfile()
+  writeLines("x", tmp)
+  googleCloudStorageR::gcs_upload(tmp, bucket = bucket, name = "x")
+  expect_true(gcp_gcs_exists(key = "x", bucket = bucket))
+})
+
+tar_test("gcp_gcs_head()", {
+  skip_if_no_gcp()
+  auth_gcp()
+  bucket <- random_bucket_name()
+  # needs to be a GCP project the tester auth has access to
+  projectId <- "gcp-setup-demo"
```

Comment:
There are more places where it would be nice to have `project_id <- Sys.getenv("GCE_DEFAULT_PROJECT_ID")`.